src/components/mapComponents/constants.tsx

```diff
@@ -26,6 +26,9 @@ export const BOSTON_BOUNDS = {
 
 export const STREET_ZOOM = 19;
 
+export const MOBILE_SLIDE_HEIGHT = 60;
+export const DESKTOP_SLIDE_HEIGHT = 60;
```

Comment:
❓ Is there a reason to distinguish the two if they're both 60?

Reply:
whoops, desktop should be 70. I'll fix that.